src/components/dialogs/name-wrapper.js

```diff
@@ -104,6 +98,14 @@ const NameWrapper = ({
         );
     };
 
+    //check if name already exist if initialValue is not empty
+    useEffect(() => {
+        if (initialValue !== '') {
+            updateFormState(initialValue);
+            setValue(initialValue);
```

Comment:
Does this effect run multiple times ? If not, there's not need for setValue as initialValue is passed to the useState() ?

Reply:
It's a bad name "initialValue" if it triggers additional behaviors when it changes. In react the convention is to only use the value for the first render

Reply:
yes, it willl trigger the useeffect twice in the case we will get the name from the backend